raster/src/main/scala/geotrellis/raster/render/ColorMaps.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2016 Azavea
```

Comment:
Since you/your company is submitting this file, this should be copyright to you or your company.